src/redux/users/usersDestroy.js

```diff
@@ -0,0 +1,75 @@
+const API_BOOKINGS_DELETE_ENDPOINT = `https://doctor-appointment-api-u7q4.onrender.com/users`;
+const REQUEST_STARTED = 'book-a-doctor/usersDestroy/REQUEST_STARTED';
+const REQUEST_FAILED = 'book-a-doctor/usersDestroy/REQUEST_FAILED';
+const REQUEST_COMPLETED = 'book-a-doctor/usersDestroy/REQUEST_COMPLETED';
+const RESET_STATE = 'book-a-doctor/usersDestroy/RESET_STATE';
+const initialState = {
+  status: 'idle',
+};
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case REQUEST_STARTED:
+      return action.payload;
+    case REQUEST_FAILED:
+      return action.payload;
+    case REQUEST_COMPLETED:
+      return action.payload;
+    case RESET_STATE:
+      return action.payload;
+    default:
+      return state;
+  }
+};
+
+const requestStarted = () => ({
+  type: REQUEST_STARTED,
+  payload: {
+    status: 'fetching',
+  },
+});
+
```

Comment:
4 (a) Add comments to describe the purpose of the reducer and the actions
   (b) Use destructuring to extract the payload from the action object in the reducer function
   (c) Rename the action types to make them more specific and descriptive